scripts/gen-edpm-kustomize.sh

```diff
@@ -199,10 +199,10 @@ cat <<EOF >>kustomization.yaml
       path: /spec/nodes/edpm-compute-${INDEX}/ansibleHost
       value: 192.168.122.$((100+${INDEX}))
     - op: replace
-      path: /spec/nodes/edpm-compute-${INDEX}/hostName
+      path: /spec/roless/edpm-compute-${INDEX}/hostName
       value: edpm-compute-${INDEX}
     - op: replace
-      path: /spec/nodes/edpm-compute-${INDEX}/openStackAnsibleEERunnerImage
+      path: /spec/roles/edpm-compute/openStackAnsibleEERunnerImage
```

Comment:
I spoke with @slagle about this yesterday because i couldn't override runner image using environment variables. It seems that the `openStackAnsibleEERunnerImage` property should be customized under service. @slagle opened https://github.com/openstack-k8s-operators/install_yamls/pull/317 to address this issue.

Reply:
We have default services. Also we can't expect users to override all services when we create an image with edpm-ansible image. IMO we should still override it per role along with per service.

Reply:
makes sense that we might need to set it on a role. the behavior (if i'm interpreting https://github.com/openstack-k8s-operators/dataplane-operator/pull/232 correctly) is that the image value on a service would override what is set on the role.